packages/tests/src/nwaku.ts

```diff
@@ -171,6 +176,8 @@ export class Nwaku {
         tcpPort,
         websocketPort,
         ...(args?.peerExchange && { discv5UdpPort }),
+        ...(isGoWaku && { minRelayPeersToPublish: 0 }),
+        ...(args?.useFilterv2 && isGoWaku && { lightClient: true }),
```

Comment:
Why does `--light-client` do in go-waku? Because we don't need go-waku to be a light client.

Reply:
This flag is not needed anymore.

Reply:
Try with :latest . I just ran the job that update that image!